api4/webhook_test.go

```diff
@@ -32,18 +32,9 @@ func TestCreateIncomingWebhook(t *testing.T) {
 
 	rhook, resp := th.SystemAdminClient.CreateIncomingWebhook(hook)
 	CheckNoError(t, resp)
-
-	if rhook.ChannelId != hook.ChannelId {
-		t.Fatal("channel ids didn't match")
-	}
-
-	if rhook.UserId != th.SystemAdminUser.Id {
-		t.Fatal("user ids didn't match")
-	}
-
-	if rhook.TeamId != th.BasicTeam.Id {
-		t.Fatal("team ids didn't match")
-	}
+	require.Equal(t, rhook.ChannelId, hook.ChannelId, "channel ids didn't match")
```

Comment:
Thanks for the submission @drekar ! 
For these tests, `require.Equal`'s signature is: `func Equal(t TestingT, expected interface{}, actual interface{}, msgAndArgs ...interface{})` -- so can you go through and make sure the `expected` and `actual` values are in the right position? 
Thanks!

Reply:
Sure thing, thanks for the review.